src/oleg.c

```diff
@@ -557,3 +559,12 @@ int ol_uptime(ol_database *db) {
     diff = difftime(now, db->meta->created);
     return diff;
 }
+
+const char *ol_last_error(const ol_database *db) {
+    static const char *human_readable[] = {
+        "No error.",
+        "Generic error."
+    };
+
+    return human_readable[db->meta->last_error];
```

Comment:
Is this safe? Maybe add a bounds check with `sizeof(human_readable)`? (try not to off-by-one)

Also I think it's a rather pedantic remark but I suspect enums aren't guaranteed to start with 0 if you don't initialize them that way. But I don't know what part of the standard says compilers are allowed to be shit in this particular way, so it might be ~cargo cult~.